examples/interactive_scripting.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import napari
+import time
+
+
+with napari.gui_qt():
+    # create the viewer with an image
+    data = np.random.random((512, 512))
+    viewer = napari.Viewer()
+    layer = viewer.add_image(data)
+
+    update_period = 0.1
+
+    def layer_update(update_interval, kwargs):
```

Comment:
```suggestion
    def layer_update(*, update_interval, num_updates):
```

Reply:
why do we accept any argument? Also i put `update_interval` and `kwargs` on purpose to demonstrate that both possible, do you still want me change @jni ?

Reply:
Typically, the best practice in Python is to use explicit argument names wherever possible, and only use `**kwargs` where necessary. When defining small functions like this one, we should follow the best practices and use named arguments.

Reply:
Oh, btw, `*,` doesn't mean accept any argument, it means, all arguments after these can only be specified as named arguments. (aka keyword-only arguments.)